src/buildmanager.cpp

```diff
@@ -2372,11 +2372,35 @@ void ProcessX::startCommand()
 		emit finished(0, NormalExit);
 		return;
 	}
+
 	if (stdoutEnabled || stdoutBuffer)
 		connect(this, SIGNAL(readyReadStandardOutput()), this, SLOT(readFromStandardOutput()));
 	if (stderrEnabled)
 		connect(this, SIGNAL(readyReadStandardError()), this, SLOT(readFromStandardError()));
 
+#ifdef Q_OS_LINUX
+	// Retrieve the environment variables
+	QProcessEnvironment env = QProcessEnvironment::systemEnvironment();
+
+	// Check if FLATPAK_SANDBOX_DIR is set

```

Comment:
Spaces shouldn't matter, since I check if `cmd_elements.first();` exists in `/app/texlive/bin`, right?

But skipping it if the command already starts with `flatpak-spawn` makes sense.

Reply:
I've now tested it with two scripts, hello.py and hello world.py, both containing this:

```
#!/usr/bin/env python3

print("Hello world")

with open("/home/alexander/success", "w") as f:
    f.write("success")
```

And it works fine if I specify `/home/alexander/hello.py` o r`"/home/alexander/hello world.py"` as a command